app/main.py

```diff
@@ -2,17 +2,20 @@
 
 import yaml
 import time
+import os
 
 import requests
 
 class Lifeguard:
     base_url = 'https://api.usmobile.com/web-gateway/api/v1'
 
     def __init__(self) -> None:
+        self.config = {}
```

Comment:
This belongs in load_config(). There are some contrived edge cases where it might even matter.